src/Dialect/ONNX/ONNXOps/Canonicalize.td

```diff
@@ -195,8 +200,8 @@ def IsNotFromONNXConstantOp: Constraint<
     "Is a value not from ONNXConstantOp">;
 
 def IsFromONNXConstantOpWithDenseElementsAttr: Constraint<
-    And<[CPred<" isa<ONNXConstantOp>($_self.getDefiningOp()) ">,
-         CPred<" onnx_mlir::getONNXConstantOp($_self).getValueAttr().isa<DenseElementsAttr>() ">
+    And<[CPred<" onnx_mlir::getONNXConstantOp($_self) ">,
+         CPred<" isa<DenseElementsAttr>(onnx_mlir::getONNXConstantOp($_self).getValueAttr()) ">
```

Comment:
I changed this because i was getting crashes with the previous implementation.  So I opted for a consistent check by using `getONNXConstantOp` in both.